### DISCUSSION TITLE
OAUTH with Azure, is documentation outdated?

### DISCUSSION THREAD

**Discussion by AndreaPux** [alephdata/aleph#2999] [General]
I asked our azure admins to create an OAUTH key/secrete for Aleph, and provided them with the instructions from Aleph documentation. 
This is their reply:

_This documentation is out of date.
On portal https://apps.dev.microsoft.com/ you get this message:
Application registrations portal is no longer available to register and manage converged applications. We recommend that you manage your existing applications and register new applications by using the App registrations (now Generally Available) experience in the Azure portal. ¨
Do you have documentation somewhere that specifically needs to be created in AZURE?_

To me is just a question of going to another section of the Azure console but I wonder if you have more specific instructions. 

Thanks!

**Comment by tillprochaska** [comment]
Hi @AndreaPux, unfortunately, we don’t use Azure AD as an identity provider at OCCRP so I’m not able to provide any additional information about it. You might be able to find someone running an Aleph instance with Azure IDP in the Aleph Slack Community: https://github.com/alephdata/aleph/blob/main/SUPPORT.md

Once you have found out how to set up Azure, it would be much appreciated if you could let us know what you had to do (either by commenting in this thread or by opening a PR to the docs)!
